jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-showcase/src/main/java/org/jbpm/console/ng/cm/server/MockCaseManagementService.java

```diff
@@ -138,12 +138,21 @@ public void destroyCaseInstance(final String serverTemplateId, final String cont
     @Override
     public void addComment(final String serverTemplateId, final String containerId, final String caseId, final String author, final String text) {
         final List<CaseCommentSummary> commentSummaryList = caseCommentMap.getOrDefault(caseId, new ArrayList<>());
-        final String id = String.valueOf(commentSummaryList.size() + 1);
+
+        int i= commentSummaryList.size() + 1;
```

Comment:
@cristianonicolai The problem is that the previous implementation could generate the same id for some comments. Conside this sequence of CRUD actions: Add, add, delete the first one, add -> there will be 2 comments with id 2.

@nmirasch I know this mock implementation is not that important, but I'd suggest using static int counter, which would be incremented every time when comment is added like so. This would make the intent to generate non-existent id cleaner:

    private static int commentIdGenerator = 0;

    public void addComment(..) {
        final String newId = String.valueOf(commentIdGenerator++);
        ...
    }
String.valueOf(commentSummaryList.size() + 1);